src/test/regress/output/appendonly.source

```diff
@@ -1776,3 +1776,44 @@ LINE 1: explain select cmax, * from test_reporting_r;
 drop table test_reporting;
 drop table test_reporting_r;
 drop table test_reporting_c;
+-- Test AO functions with execution location on dispatcher.
+drop table if exists t1;
+drop table if exists t2;
+drop table if exists d;
+create table t1 (a int) with (appendoptimized=true, orientation=row);
+create table t2 (a int) with (appendoptimized=true, orientation=column);
+create table d (tname text);
```

Comment:
Maybe, rename this table to "tables_list".

Reply:
fixed